layout/lib/practice/bottom_tabs.dart

```diff
@@ -0,0 +1,58 @@
+import 'package:flutter/material.dart';
+
+class BottomTabs extends StatelessWidget {
+  @override
+  Widget build(BuildContext context) {
+    return Container(
+      child: Column(children: <Widget>[
+        Container(
+          height: 70,
+          child: Stack(
+            textDirection: TextDirection.ltr,
```

Comment:
text direction is generally not necessary and should use a directionality provided from Material/CupertinoApp